locale/en.json

```diff
@@ -40,11 +40,13 @@
     }
   },
   "taskList": {
-    "header": "Your PIP benefit appeal",
+    "header": "Your benefit appeal",
```

Comment:
Instead of creating different heading you can add benefitType as acronym 
e.g.
"oral": [
          "The hearing for your {{appeal.benefitType | agencyAcronym}}

Reply:
This didn't work. Other pages use this header (not as a title) that don't have the appeal in scope, which is why I kept a default header value. I can pair on this with you if you like. 

There were issues with keeping the title in session which is why I took the approach of combining the text.

I tried adding the following in status.ts
```
  const { renderContent } = require('../../core/tyaNunjucks');
  const i18n = require('../../../locale/en.json');
  const taskListContent = i18n.taskList;
  const placeholder = { benefitType: req.session.hearing.appeal_details.benefit_type };
  i18n.taskList = renderContent(taskListContent, placeholder);
```
It was no good as it kept the title in session, when the user swapped to other cases they were on it didn't update the title.